### DISCUSSION TITLE
Help with Development of jtl-reporter

### DISCUSSION THREAD

**Discussion by kamal2222ahmed** [ludeknovy/jtl-reporter#51] [General]
Hi @ludeknovy 
Is it possible to contribute to the development of jtl-reporter? I can continue as Tester/Quality Assurance member, but if there is a low hanging fruit/task, i can help with.
Thanks,
-Kamal.

**Comment by ludeknovy** [comment]
Hi @kamal2222ahmed!

Of course it is possible! There is a couple of tasks in the issues, but I am not sure If they could be called low hanging fruit.
It depends on your skillset. But let me go through my personal backlog and see whether I could find some "good first issue".

Oh, and the documentation could be improved as well.

Btw. you made the project better already by submitting the issues and feature request ;)

**Reply by ludeknovy** [reply]
I think this one could be classified as low hanging fruit:  https://github.com/ludeknovy/jtl-reporter/issues/52
@kamal2222ahmed
